src/components/Modals/ExpertOverviewModal.tsx

```diff
@@ -86,12 +86,27 @@ export const ExpertOverviewModal: React.FC = () => {
     >
       <div className={classes.modal}>
         <Title size="h2">{MODAL_TITLE}</Title>
-        <Search
-          placeholder="Suche nach Expert*innen für.."
-          allowClear
-          onSearch={onSearch}
-          className={classes.search}
-        />
+        {experts && (
```

Comment:
Why do you check `experts` here?

Reply:
It was a check for testing. I removed it